lib/engine/game/g_1822/step/tracker.rb

```diff
@@ -78,6 +78,13 @@ def remove_border_calculate_cost!(tile, entity_or_entities, spender)
             next 0 unless hex.targeting?(neighbor)
 
             tile.borders.delete(border)
+
+            # delete border on adjacent hex side if it also has track, i.e., if
+            # it was preprinted on a gray hex
+            n_tile = neighbor.tile
+            n_edge = hex.invert(edge)
+            n_tile.borders.reject! { |nb| nb.edge == n_edge && n_tile.exits.include?(n_edge) }
```

Comment:
` n_tile.borders.reject! { |nb| nb.edge == n_edge } if n_tile.exits.include?(n_edge)` is probably a little more efficient.